crates/crypto/src/key/key.rs

```diff
@@ -2,6 +2,13 @@ use ssi_jwk::JWK;
 use ssi_jws::Error as JWSError;
 
 /// Enum defining all supported cryptographic key types.
+///
+/// Note that KeyType is NOT to be interpreted as the "kty" header defined in https://www.rfc-editor.org/rfc/rfc7517#section-4.1.
```

Comment:
this note is not necessary since this enum is not defined in the context of JOSE